src/modules/familyMember/application/familyMember.service.ts

```diff
@@ -31,29 +32,26 @@ export class FamilyMemberService {
 
         const existingByReg = await this.familyMemberRepository.findByRegistrationNumber(dto.registrationNumber);
         if (existingByReg) {
-            throw new ConflictException(`CPF '${dto.registrationNumber}' already in use!`);
+            throw new ConflictException(`CPF '${dto.registrationNumber}' is already in use.`);
         }
 
         if (dto.email) {
             const existingMember = await this.familyMemberRepository.findByEmail(dto.email);
             if (existingMember) {
-                throw new ConflictException(`Email '${dto.email}' already in use!`);
+                throw new ConflictException(`Email '${dto.email}' is already in use.`);
             }
         }
         
         if (dto.addressId !== undefined) {
```

Comment:
```suggestion
        if (dto.addressId != undefined) {
```
Ao se passar um valor como:
```
{
  "fullName": "Joana Oliveira",
  "relationship": "Mãe",
  "phoneNumber": "51999999999",
  "studentIds": [
    2
  ],
  "addressId": null,
  "email": "joana.o@email.com",
  "socialName": "Joana",
  "race": "BRANCA",
  "gender": "MASCULINO",
  "educationLevel": "NENHUM",
  "dateOfBirth": "1990-09-11",
  "socialPrograms": "BOLSA_FAMILIA",
  "employmentStatus": "EMPREGADO",
  "nis": "12345678900",
  "registrationNumber": "38224936864"
}
```
Em que o `addressId` é nulo, a aplicação está quebrando.